#! /usr/bin/python

PACKAGE = "blackbox"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("roi_lower_x", double_t, 0, "ROI: Lower boundary on X-axis", 0.03, -1.0, 0.05)
gen.add("roi_upper_x", double_t, 0, "ROI: Upper boundary on X-axis", 0.27, 0.05, 1.0)
gen.add("roi_lower_y", double_t, 0, "ROI: Lower boundary on Y-axis", 0.03, -1.0, 0.05)
gen.add("roi_upper_y", double_t, 0, "ROI: Upper boundary on Y-axis", 0.27, 0.05, 1.0)
gen.add("roi_lower_z", double_t, 0, "ROI: Lower boundary on Z-axis", -0.20, -1.0, -0.05)
gen.add("roi_upper_z", double_t, 0, "ROI: Upper boundary on Z-axis", 0.0, -0.05, 1.0)

exit(gen.generate(PACKAGE, "blackbox", "Blackbox"))
